src/server/components/lists/controllers/ingest/ingestPutController.ts

```diff
@@ -43,6 +43,8 @@ export async function ingestPutController(req: Request, res: Response) {
       list: {
         select: {
           jsonData: true,
+          type: true,
+          country: true,
```

Comment:
I don't think you need list.type - `listItem.type` should be available

Reply:
Unfortunately it looks like type doesn't get passed along 'typeSingular' in blank 👇 😢

<img width="1135" alt="Screenshot 2023-06-06 at 16 58 13" src="https://github.com/UKForeignOffice/lists/assets/1377253/b07b37ca-961b-40b9-bdd9-e8b54478b8b0">